tests/integration/security/rbac/util/rbac_util.go

```diff
@@ -86,3 +89,17 @@ func (tc TestCase) CheckRBACRequest() error {
 	}
 	return nil
 }
+
+func RunRBACTest(t *testing.T, cases []TestCase) {
+	for _, tc := range cases {
+		testName := fmt.Sprintf("%s->%s:%s%s[%v]",
```

Comment:
The last two %s%s do not have a delimiter between them; I am not sure how to tell them apart. I recall I have commented the non-delimited %s%s before in a previous PR but a delimiter is not added.
@nmittler @ozevren Do you think a delimiter should be added between %s%s?

Reply:
Previous conversation: https://github.com/istio/istio/pull/13896#discussion_r281862139

Path already has "/", if we add any character between the `%s%s`, it won't look as clean as it is to me.

cc @yangminzhu

Reply:
What if path does not already has "/"?

Reply:
Morever, if PortName also has "/", how to distinguish PortName and Path?